.gitignore

```diff
@@ -1,7 +1,7 @@
 # Content directory
-content/inputs/
-content/outputs/intermediates/
-content/outputs/new_prompts.txt
+tracex/extraction/content/inputs
```

Comment:
Isn't the content folder seperate to the traxec django project folder?

Reply:
Django really doesn't like accessing files from outside its own scope. So we moved it inside the scope.